ibutex.py

```diff
@@ -53,8 +57,35 @@
 			path = os.path.realpath("../{}".format(f))
 			os.symlink(path, f)
 
+def get_inputs(root_tex_file):
+	results = []
+	with open(root_tex_file, "r") as f:
+		tex_data = f.read()
+		re_search = re.compile("\\\\input\\{(.*)\\}")
+		m = re_search.findall(tex_data)
+		for r in m:
+			if os.path.isfile(r+".tex"):
+				results.append(r+".tex")
+	return results
+
+inputs = [ "../"+texfile ]
+if not args.spelling_lang is None:
+	i = 0
+	while i < len(inputs):
+		for j in get_inputs(inputs[i]):
+			inputs.append(j)
+	i += 1
+
+if not args.spelling_lang is None:
```

Comment:
I don't think that ibutex is the right place for spell checking, see also the other PR for details.

Reply:
okay i will update this PR and remove the spell checking